packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -0,0 +1,140 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { BadgeState } from './Badge.types';
+
+/**
+ * Styles for the root slot
+ */
+export const useRootStyles = makeStyles<BadgeState>([
+  [
+    null,
+    theme => ({
+      display: 'inline-flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: theme.alias.color.brand.brandBackground,
+      borderColor: theme.alias.color.brand.brandBackground,
+      color: theme.global.color.white,
+      fontWeight: theme.global.type.fontWeights.semibold,
+      borderWidth: '1px',
+      borderStyle: 'solid',
+      fontFamily: theme.global.type.fontFamilies.base,
+    }),
+  ],
+  [
+    s => s.size === 'smallest',
+    theme => ({
+      width: '6px',
+      height: '6px',
+      fontSize: '4px',
+    }),
+  ],
+  [
+    s => s.size === 'smaller',
+    theme => ({
+      width: '10px',
+      height: '10px',
+      fontSize: '6px',
+    }),
+  ],
+  [
+    s => s.size === 'small',
+    theme => ({
+      minWidth: '4px',
+      height: '16px',
+      paddingRight: '6px',
+      paddingLeft: '6px',
+      gap: '4px',
+      fontSize: '8px',
+    }),
+  ],
+  [
+    s => s.size === 'medium',
+    theme => ({
+      height: '20px',
+      minWidth: '4px',
+      gap: '4px',
+      paddingRight: '8px',
+      paddingLeft: '8px',
+      fontSize: '12px',
+    }),
+  ],
+  [
+    s => s.size === 'large',
+    theme => ({
+      minWidth: '8px',
+      height: '24px',
+      paddingRight: '8px',
+      paddingLeft: '8px',
+      fontSize: '12px',
+      gap: '6px',
+    }),
+  ],
+  [
+    s => s.size === 'larger' || s.size === 'largest',
+    theme => ({
+      minWidth: '8px',
+      height: '32px',
+      paddingRight: '12px',
+      paddingLeft: '12px',
+      gap: '6px',
+      fontSize: '12px',
+      border: '2px',
```

Comment:
Should be `borderWidth` instead of `border`, and use a token:
`borderWidth: theme.global.strokeWidth.thick`